_includes/description.html

```diff
@@ -1,6 +1,11 @@
 <h2>Meet the Pod</h2>
 <div class="card-container">
     {% for item in site.data.fellows %}
+    {% if item.page-name %}
+    <a href="/description/{{ item.page-name }}">
+    {% else %}
+    <a>
```

Comment:
Similar issue here. If there is no `{{item.page-name}}` what do you think should be rendered here instead?

Reply:
i think it should be something that would lead to different personalized pages for each particular fellow instead of one overall description page. 
This is my first time working with ruby so am not that sure on how to proceed

Reply:
Maybe if there is no `{{item.page-name}}`, maybe there can be a message stating there is no description or leave it blank

Reply:
I created separate markdown files for each of the fellows so that each participant can have personalized pages with a unique path.
![Screenshot (97)](https://user-images.githubusercontent.com/56563537/157460408-1f649c6e-c742-4de1-8ae0-693863d3a86f.png)
![Screenshot (98)](https://user-images.githubusercontent.com/56563537/157460418-eb093771-1a91-404f-8aaf-03f187295d89.png)
![Screenshot (99)](https://user-images.githubusercontent.com/56563537/157460424-3dacd0dc-191d-4725-ba27-7658cf0e392d.png)
![Screenshot (100)](https://user-images.githubusercontent.com/56563537/157460433-0b5033ab-2676-409c-9de1-d53b489649ed.png)